charts/fybrik/templates/opa-server/opa-deployment.yaml

```diff
@@ -72,6 +72,13 @@ spec:
             EOF
             {{ end }}
             {{ end }}
+            {{- if .Values.opaServer.defaultDecision}}
+            {{- range $policyName, $policy := .Values.opaServer.defaultDecision }} 
+            {{- printf "\ncat > /bootstrap/%s.rego <<EOF" $policyName  | indent 12 }}
+            {{ $policy | indent 12 }}
+            EOF
+            {{ end }}
```

Comment:
Shouldn't `charts/fybrik/files/opa-server/policy-lib/default_policy.rego` be updated?

Reply:
@revit13 No, but maybe the name is misleading. This file is used for an internal implementation of opa-connector, maybe something like lib.rego or internal.rego

Reply:
@shlomitk1 thanks just updated my comment. I think that adding an explanation for the verdict will be helpful.